pkg/api/controllers/workspace/toolbox/git.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package toolbox
+
+import "github.com/gin-gonic/gin"
+
+// GitAddFiles			godoc
+//
+//	@Tags			workspace toolbox
+//	@Summary		Add files
+//	@Description	Add files to git commit
+//	@Produce		json
+//	@Param			workspaceId	path	string			true	"Workspace ID or Name"
+//	@Param			projectId	path	string			true	"Project Id"
+//	@Param			params		body	GitAddRequest	true	"GitAddRequest"
+//	@Success		200
+//	@Router			/workspace/{workspaceId}/{projectId}/toolbox/git/add [post]
+//
+//	@id				GitAddFiles
+func GitAddFiles(ctx *gin.Context) {
+	forwardRequestToToolbox(ctx)
+}
+
+// GitCloneRepository			godoc
+//
+//	@Tags			workspace toolbox
+//	@Summary		Clone git repository
+//	@Description	Clone git repository inside workspace project
+//	@Produce		json
+//	@Param			workspaceId	path	string			true	"Workspace ID or Name"
+//	@Param			projectId	path	string			true	"Project Id"
+//	@Param			params		body	GitCloneRequest	true	"GitCloneRequest"
+//	@Success		200
+//	@Router			/workspace/{workspaceId}/{projectId}/toolbox/git/clone [post]
+//
+//	@id				GitCloneRepository
+func GitCloneRepository(ctx *gin.Context) {
+	forwardRequestToToolbox(ctx)
+}
+
+// GitCommitChanges			godoc
+//
+//	@Tags			workspace toolbox
+//	@Summary		Commit changes
+//	@Description	Commit changes to git repository inside workspace project
+//	@Produce		json
+//	@Param			workspaceId	path		string				true	"Workspace ID or Name"
+//	@Param			projectId	path		string				true	"Project Id"
+//	@Param			params		body		GitCommitRequest	true	"GitCommitRequest"
+//	@Success		200			{object}	GitCommitResponse
+//	@Router			/workspace/{workspaceId}/{projectId}/toolbox/git/commit [post]
+//
+//	@id				GitCommitChanges
+func GitCommitChanges(ctx *gin.Context) {
+	forwardRequestToToolbox(ctx)
+}
+
+// GitCreateBranch			godoc
+//
+//	@Tags			workspace toolbox
+//	@Summary		Create branch
+//	@Description	Create branch on git repository inside workspace project
+//	@Produce		json
+//	@Param			workspaceId	path	string				true	"Workspace ID or Name"
+//	@Param			projectId	path	string				true	"Project Id"
+//	@Param			params		body	GitBranchRequest	true	"GitBranchRequest"
+//	@Success		201
+//	@Router			/workspace/{workspaceId}/{projectId}/toolbox/git/branchs [post]
```

Comment:
![image](https://github.com/user-attachments/assets/e2e278ad-c9de-492e-94c4-258614c1b599)

"branches" instead of "branchs"

Reply:
Amended into https://github.com/daytonaio/daytona/pull/1455/commits/3dad154f98db566404d4931fc7c36baf4604d304